client/src/js/components/bhHiddenField.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @overview bhHiddenField
+ *
+ * @description
+ * This component allows a programmer to optionally hide a field in a form at
+ * startup that may toggled open by the user if they need it.
+ */
+
+var template =
+ '<p>' +
```

Comment:
This is so small but since we are not using ECMA 6 syntax to make string templating at the client side, you can create a separate file to make it easy to read.